src/expr/src/expr/expr_in.rs

```diff
@@ -142,7 +142,7 @@ mod tests {
     #[test]
     fn test_in_expr() {
         let input_ref_expr_node = ExprNode {
-            expr_type: Type::InputRef as i32,
+            function_type: Type::Unspecified as i32,
```

Comment:
Seems we would need `#[expect(clippy::foo)]` (or `allow`) after #8367 is in effect.